### DISCUSSION TITLE
Notifications when updating

### DISCUSSION THREAD

**Discussion by driver288** [Romanitho/Winget-AutoUpdate#654] [Ideas]
Hi!

Nice project this! I'm testing it out at the moment. In the project start page there is an image of a notification showint an app being updated. When i use the updater i get buttons in the UI of the notification for View Logs and turn off. i don't want users to see these buttons but only be informed that something is happening. I use the Microsoft Store version for use with intune as we use it for many customers and install/uninstall apps that way. Is there a way to make these buttons disapear?

**Comment by KnifMelti** [comment]
If this is what you mean it's because the user have triggered an update manually via a shortcut, then it shows (because the user can be interested in seeing the log):
![image](https://github.com/user-attachments/assets/e22b4f94-027d-48bb-ba45-8c745703bba9)

When automatically started via the scheduled task `Winget-AutoUpdate` the only thing an user sees is depending on what setting you have (**Full, Success only or none**)
https://github.com/Romanitho/Winget-AutoUpdate?tab=readme-ov-file#notification-level
Then only the notifications of the updates of apps is shown (as in the image of the project start page your referring to) if there are any updates.
With `the Microsoft Store version` you mean a seperate fork (https://github.com/Weatherlights/Winget-AutoUpdate-Intune) not this repo?
Then it's a standalone fork.

**Comment by whitephnx1** [comment]
On this same subject, is there a way to get success only to show the initial warning of the app being updated? we want users to know why their app suddenly closed, but then only want to warn them if it succeeded after that. bc we have some apps that are failing bc winget doesn't know how to kill its processes to update it so it just fails. we just don't want any unnecessary alarm of Failure going off. just need to show success, but warn that it is going to start the update process before that.

**Reply by Romanitho** [reply]
There is no notification on WAU start when it is automatically started. To much information then on a daily basis run for example.
